src/property.ts

```diff
@@ -148,9 +148,7 @@ export class Property implements PropertyPath {
 				// Specifier
 				if (typeof (options.format) === "string") {
 					let format = options.format;
-					targetType.model.ready(() => {
-						this.format = targetType.model.getFormat(this.propertyType, format);
-					});
+					this.format = targetType.model.getFormat(this.propertyType, format);
```

Comment:
Immediately create and assign the format to the property when extending the model. This means when the model is ready, all properties that should have formats defined will have a value for the `format` property.